src/webviews/apps/settings/partials/modes.html

```diff
@@ -2,7 +2,7 @@
 	<div class="section__header">
 		<h2>
 			Modes
-			<a class="link__learn-more" title="Learn more" href="https://github.com/gitkraken/vscode-gitlens/#modes-">
+			<a class="link__learn-more" title="Learn more" href="https://help.gitkraken.com/gitlens/settings/#modes-settings">
```

Comment:
Important note: We don't seem to have a section in the Help Center for Modes (other than the one in Settings which specifies settings variables but doesn't overview the feature). This needs to be updated once we have that section.